lib/AppInfo/Application.php

```diff
@@ -89,8 +89,8 @@ public function registerNavigation(IUserSession $userSession): void {
 		if ($user !== null) {
 			$userId = $user->getUID();
 			$container = $this->getContainer();
-
-			if ($this->config->getUserValue($userId, self::APP_ID, 'navigation_enabled', '0') === '1') {
+			$navlink_default = $this->config->getAppValue(Application::APP_ID, 'navlink_default');
+			if ($this->config->getUserValue($userId, self::APP_ID, 'navigation_enabled', $navlink_default) === '1') {
```

Comment:
`$navlinkDefault` would be better suited to be consistent with camel case naming style